include/socgraph.php

```diff
@@ -1244,18 +1244,20 @@ function poco_discover_federation() {
 			poco_check_server("https://".$server->host);
 	}
 
-	// Discover GNU Social Servers
-	if (!get_config('system','ostatus_disabled')) {
-		$serverdata = "http://gstools.org/api/get_open_instances/";
-
-		$result = z_fetch_url($serverdata);
-		if ($result["success"]) {
-			$servers = json_decode($result["body"]);
-
-			foreach($servers->data AS $server)
-				poco_check_server($server->instance_address);
-		}
-	}
+	// Currently disabled, since the service isn't available anymore.
```

Comment:
This change is now causing a merge conflict when I try to rebase my develop on it. Please rewrite it to a multi-line comment as I have suggested before it is being merged to `develop`.